test/e2e/webdriver/driver.js

```diff
@@ -6,16 +6,25 @@ const {
   error: webdriverError,
   Key,
   until,
+  ThenableWebDriver, // eslint-disable-line no-unused-vars -- this is imported for JSDoc
+  WebElement, // eslint-disable-line no-unused-vars -- this is imported for JSDoc
 } = require('selenium-webdriver');
 const cssToXPath = require('css-to-xpath');
 const { retry } = require('../../../development/lib/retry');
 
+const PAGES = {
+  BACKGROUND: 'background',
+  HOME: 'home',
+  NOTIFICATION: 'notification',
+  POPUP: 'popup',
+};
+
 /**
  * Temporary workaround to patch selenium's element handle API with methods
  * that match the playwright API for Elements
  *
  * @param {object} element - Selenium Element
- * @param driver
+ * @param {!ThenableWebDriver} driver
```

Comment:
Oh interesting. I've never seen the `!` used like this. I would think that non-nullable would be the default, but maybe not? 🤷🏻

Reply:
I didn't discover it and I wasn't the first to use it here -- it was already in chrome.js, firefox.js, and driver.js

But the default in JavaScript is always that things can be null/undefined